tests/functional/t/typevar_naming_style_default.py

```diff
@@ -16,12 +16,12 @@
 GoodBoundNameT = TypeVar("GoodBoundNameT", bound=int)
 
 # Some of these will create a RunTime error but serve as a regression test
-T = TypeVar(  # [typevar-name-incorrect-variance]
+T = TypeVar(  # [typevar-double-variance]
```

Comment:
I wondered what we should do in this case. I don't really like how we suppress the `name-incorrect` warning here. Ie., if a user removes on of the variances they would need to do a new pylint run to see that name is still incorrect.
For users that rely on CI to run `pylint` this can be frustrating as it might take some time for that message to reach them.
Can we think of a way to display both messages?

Reply:
I agree we should emit both messages.

Reply:
I actually removed this message intentionally because it's not really appropriate to suggest a correct name in these double invariance cases since we don't know which type of variance they're intending.

Referring to current main branch:
<img width="1045" alt="image" src="https://user-images.githubusercontent.com/80432516/160589910-2b8e396d-5aec-45b5-9857-ddc1cbc75d0e.png">

The only reason we were suggesting "T_co" is just because the old code saw `covariant=True` and immediately returned, but you can't tell here if the user actually wants "T_contra"?

These examples below aren't necessarily wrong names (actually in context they're likely correct). We need the double-variance to be addressed first before we can say whether they are wrong or not and therefore why distract from the user with a false positive wrong name?
```python
T_co = TypeVar("T_co", covariant=True, contravariant=True)
T_contra = TypeVar("T_contra", covariant=True, contravariant=False)
```

Reply:
I understand that issue, but I don't like how we need two runs to fix all linting issues.

Can we do something like `Type variable name does not reflect variance.` and then pass `"T" is covariant, use "T_co" instead` as arg to the message if we know the expected variance?

Reply:
Yeah i'm happy to do the generic message approach in this case then 👍 

(just noting details below for future ref)
- generic message prompts the user to consider fixing the typevar name as well as double-variance. Strictly speaking they would still have to run the linter a second time after fixing double-variance to get a suggestion on what exactly to change the name to, but users familiar with the naming convention will likely know to check/fix the name if needed after the first generic message.
```python
T = TypeVar("T", covariant=True, contravariant=True)
```

- Wording of `Type variable name does not reflect variance.` is correct here, even if they do probably want this name. If they fix the double-invariance and the name turns out to be correct then the message wouldn't show up in subsequent runs so no overhead of prompting the user to consider potential bad naming.
```python
T_co = TypeVar("T_co", covariant=True, contravariant=True)
```